BetterStats/BetterStats.cs

```diff
@@ -113,8 +113,24 @@ private class ProductMetrics
         private static void ClearEnhancedUIProductEntries()
         {
             if (statWindow == null) return;
+
+            foreach (EnhancedUIProductEntryElements enhancement in enhancements.Values)
```

Comment:
I didn't realize the labels were overlaying each other when the plugin was reloaded by script engine. This should resolve that issue